src/alternative_interfaces.jl

```diff
@@ -173,7 +173,7 @@ function fzero(
 end
 
 function fzero(f, bracket::Tuple{T,S}; kwargs...) where {T<:Number,S<:Number}
-    d = Dict(kwargs)
+    d = Dict(kwargs...)
```

Comment:
IIRC an alternative without splatting could be

```suggestion
    d = Dict(pairs(kwargs))
```